python/orca/src/bigdl/orca/data/ray_xshards.py

```diff
@@ -208,14 +208,8 @@ def to_spark_xshards(self) -> "SparkXShards":
         password = ray_ctx.redis_password  # type: ignore
         partition2store = self.partition2store_name
         rdd = self.rdd.mapPartitionsWithIndex(
-            lambda idx, _: get_from_ray(idx, address, password, partition2store))
-
-        # the reason why we trigger computation here is to ensure we get the data
-        # from ray before the RayXShards goes out of scope and the data get garbage collected
```

Comment:
After adding lifetime=detached, the actors wouldn't be gc'ed and thus the logic below is not needed.